conda_smithy/linters_forge_yml.py

```diff
@@ -0,0 +1,87 @@
+import json
```

Comment:
Can you rename this file to `lint_forge_yml.py`?

Reply:
done